history-of-quakers/modernized/02-preface.adoc

```diff
@@ -314,10 +314,10 @@ yet gave a good account and favorable testimony concerning them,
 as may be seen in Richard Claridge`'s answer to a book of Edward Cockson, page 266,
 and seq.
 And at Amsterdam in Holland, many years ago, a learned man published a book called,
-Lucerna super Candelabrum,
+_Lucerna Super Candelabrum,_
 wherein he very eminently defended the doctrine of the inward light;
 and this book was published in Dutch, and afterwards also into English,
-with the title of The Light upon the Candlestick: and since the name of William Ames,
+with the title of _The Light Upon the Candlestick:_ and since the name of William Ames,
```

Comment:
I had to look this up, but _upon_ is a preposition, so it's not capitalized in titles.  So probably also change the _Super_ above on line 317 back as well. 👍